Apple/__main__.py

```diff
@@ -312,11 +312,18 @@ def unpack_deps(
     On iOS, as a safety mechanism, any dynamic libraries will be purged from
     the unpacked dependencies.
     """
+    # To create new builds of these dependencies, usually all that's necessary
+    # is to push a tag to the cpython-apple-source-deps repository, and GitHub
+    # Actions will do the rest.
+    #
+    # If you're a member of the Python core team, and you'd like to be able to
+    # push these tags yourself, please contact Malcolm Smith or Russell
+    # Keith-Magee.
```

Comment:
If you're looking for an increase of bus factor for these, I'm happy to oblige :)

Reply:
I've sent you an invite to both the Apple and Android repos. Hugo, Gregory and Erlend already had access to the Android repo.